### ISSUE TITLE
[React Native Gallery Preview]: 'React Native Gallery Preview' text present in title bar is not adapting to 'Dark Mode'

### ISSUE THREAD

**Issue by protikbiswas100** [microsoft/react-native-gallery#676]
https://microsoft.visualstudio.com/OS/_workitems/edit/59203114
Build Details:
OS Version: 24H2 
OS Build: 26100.5074
App: React Native Gallery Preview
AT Used: Dark Mode

Pre-Requisite:
Settings > Personalization > Color > Dark.

Repro-Steps:
Launch the React Native Gallery-Preview app.
Now observe the 'React Native Gallery Preview' text present in title bar
Actual Result:
'React Native Gallery Preview' text present in title bar is not adapting to 'Dark Mode'

Note: 'Dark mode' is not adapting whole 'Title bar' section

Expected Result:
'React Native Gallery Preview' text present in title bar is should be adapt to the 'Dark Mode'

User Impact:
Low vision or color blindness users who depend on dark mode will face difficulty, if dark theme is not adapting to 'React Native Gallery Preview' text present in title bar

MAS Reference Link: [MAS 4.3.1 – No Disruption of Accessibility Features](https://aka.ms/MAS4.3.1)

Trap ID: [9.1 - Unattractive Appearance](https://aka.ms/UnattractiveAppearance)

“Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”
